system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -3,16 +3,13 @@
 
     .click-target {
       cursor: pointer;
+      vertical-align: top;
     }
 
-    .icon {
-      &.icon-arrowhead-down {
-        margin: -3px 0 0 8px;
-      }
-
-      &.icon-arrowhead-forward {
-        margin: -2px 0 0 8px;
-      }
+    .icon-arrowhead-down,
+    .icon-arrowhead-forward {
+      margin-top: -1px;
+      margin-inline-start: 8px;
```

Comment:
from slack re transition:
> I think it’s a nice touch

let's do it! same animation timing as the section content hiding/showing. Probably add it to this style block. Need to fix up the rotate degs to be the appropriate shortest path. Probably 90deg and -90deg for the appropriate rtl/ltr.